client/api/two/webhooks.py

```diff
@@ -0,0 +1,48 @@
+from client.resource import Resource
+
+
+class Webhooks(Resource):
+
+    def all(self, page=None, limit=20):
+        params = {
+            'page': page,
+            'limit': limit
+        }
+        return self.request_get('webhooks', params)
+
+    def create(self, event, target, conditions, enabled=None):
+        data = {
+            'event': event,
+            'target': target,
+            'conditions': conditions
+        }
+        if enabled is not None:
+            data.update({
+                'enabled': enabled
+            })
+        return self.request_post('webhooks', data)
+
+    def get(self, webhook_id):
+        return self.request_get('webhooks/{}'.format(webhook_id))
+
+    def update(
+            self,
+            webhook_id,
+            event=None,
+            target=None,
+            conditions=None,
+            enabled=None
+    ):
+        data = {}
+        if event is not None:
```

Comment:
you can just do `if event:`

Reply:
I usually opt not to, as that also conditions on the truth value of `event` itself, even when not `None`. You could make an argument that these fields should always be truthy when not `None`, though (except, of course, for `enabled` ;)). So I'll trust your judgement and lop off the `is not None`.

Reply:
Addressed the top two cases, left `enabled` untouched (for reason given in the previous comment), as well as `conditions` (for the `[]` case).